docs/csharp/programming-guide/xmldoc/how-to-use-the-xml-documentation-features.md

```diff
@@ -11,58 +11,63 @@ The following sample provides a basic overview of a type that has been documente
   
 ## Example  
  [!code-csharp[csProgGuideDocComments#15](../../../csharp/programming-guide/xmldoc/codesnippet/CSharp/how-to-use-the-xml-documentation-features_1.cs)]  
-  
- **// This .xml file was generated with the previous code sample.**  
-**\<?xml version="1.0"?>**  
-**\<doc>**  
- **\<assembly>**  
- **\<name>xmlsample\</name>**  
- **\</assembly>**  
- **\<members>**  
- **\<member name="T:SomeClass">**  
- **\<summary>**  
- **Class level summary documentation goes here.\</summary>**  
- **\<remarks>**  
- **Longer comments can be associated with a type or member**  
- **through the remarks tag\</remarks>**  
- **\</member>**  
- **\<member name="F:SomeClass.m_Name">**  
- **\<summary>**  
- **Store for the name property\</summary>**  
- **\</member>**  
- **\<member name="M:SomeClass.#ctor">**  
- **\<summary>The class constructor.\</summary>**  
- **\</member>**  
- **\<member name="M:SomeClass.SomeMethod(System.String)">**  
- **\<summary>**  
- **Description for SomeMethod.\</summary>**  
- **\<param name="s"> Parameter description for s goes here\</param>**  
- **\<seealso cref="T:System.String">**  
- **You can use the cref attribute on any tag to reference a type or member**  
- **and the compiler will check that the reference exists. \</seealso>**  
- **\</member>**  
- **\<member name="M:SomeClass.SomeOtherMethod">**  
- **\<summary>**  
- **Some other method. \</summary>**  
- **\<returns>**  
- **Return results are described through the returns tag.\</returns>**  
- **\<seealso cref="M:SomeClass.SomeMethod(System.String)">**  
- **Notice the use of the cref attribute to reference a specific method \</seealso>**  
- **\</member>**  
- **\<member name="M:SomeClass.Main(System.String[])">**  
- **\<summary>**  
- **The entry point for the application.**  
- **\</summary>**  
- **\<param name="args"> A list of command line arguments\</param>**  
- **\</member>**  
- **\<member name="P:SomeClass.Name">**  
- **\<summary>**  
- **Name property \</summary>**  
- **\<value>**  
- **A value tag is used to describe the property value\</value>**  
- **\</member>**  
- **\</members>**  
-**\</doc>**   
+
+The example generates an .xml file with the following contents:
+
+```xml  
+ // This .xml file was generated with the previous code sample.  
```

Comment:
That line can be removed as the line above introduces the file contents.

Reply:
Thanks for noticing that, @pkulikov! I'd added the introductory line above so that this line could be removed -- and then I forgot to remove it.